### PR TITLE
updated selector to use vite

### PR DESCRIPTION
**Pull request by Adamws33**
Updated the selector to use vite.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,20 +29,21 @@
...
     "react-swipeable-views-react-18-fix": "^0.14.1",
     "react-swipeable-views-utils": "^0.14.0",
     "redux": "^4.2.1",
     "redux-devtools-extension": "^2.13.9",
     "sass-mediaqueries": "^1.6.1",
+    "shared-react-components": "github:precision-sustainable-ag/shared-react-components",
     "styled-components": "^5.3.6",
-    "worker-loader": "^3.0.8"
+    "worker-loader": "^3.0.8",
+    "xlsx": "^0.18.5"
```
Comment (mikahpinegar) [inline package.json:40]: is this needed?

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://57.152.77.147:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://57.152.77.147:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://57.152.77.147:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://57.152.77.147:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by YazidZidane** [comment]
Can you also rename all the `mapBoxToken` to `mapboxToken`? I have updated it in shared components just to have a clearer naming.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://57.152.77.147:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://57.152.77.147:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by YazidZidane — APPROVED** [review]

**Pull request merged**